<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDialog</class>
 <widget class="QDialog" name="ChatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButton_send">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_connect">
      <property name="text">
       <string>connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidget_online">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>131</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
